### PR TITLE
TF-1740: Fix double vertical scroll bar on composer

### PR DESCRIPTION
**Pull request by nqhhdev**
- Issues: #1740 

- Resolved :

- Need to merge  https://github.com/linagora/html-editor-enhanced/pull/11

`Mac OS`
https://user-images.githubusercontent.com/99852347/233457448-63800f06-df1c-4086-8349-51e99b42999d.mov

`Linux`

https://user-images.githubusercontent.com/99852347/233553849-b6319665-e72e-4c63-845d-df7ae5dcba49.mov

https://user-images.githubusercontent.com/99852347/233553903-27866b50-a24f-428b-a4de-69cfbf61e66f.mov

https://user-images.githubusercontent.com/99852347/233554183-11c569a9-b158-4c62-905f-592d849fbe36.mov

`Code View` Mode

https://user-images.githubusercontent.com/99852347/233694156-33cfa9bb-b329-42c7-91fe-667520842d58.mov

### CODE REVIEW

File: docs/adr/0022-fix-double-vertical-scroll-bar-on-composer.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+## Context
+
+- Root cause: 
+    - When we using `html_editor_enhanced` lib can't auto resize `HTML Iframe` when changing the size of the screen
+    - `.note-editor.note-airframe .note-editing-area .note-editable,.note-editor.note-frame .note-editing-area .note-editable` has an overridden padding under the widgets
+
+## Decision
+
+- Use `setFullScreen` to `HTML Iframe` for the purpose of automatically resizing when there are changes
+- Remote padding of `.note-editor.note-airframe .note-editing-area .note-editable,.note-editor.note-frame .note-editing-area .note-editable`
```
Comment (dab246) [inline docs/adr/0022-fix-double-vertical-scroll-bar-on-composer.md:18]: Should it be `Remove`?

**Comment by dab246** [comment]
IMO, you should add these 2 changed values for what to `adr` so we can take note of this issue.

Screenshot 2023-04-21 at 15 14 48

Screenshot 2023-04-21 at 15 15 12

File: docs/adr/0022-fix-double-vertical-scroll-bar-on-composer.md (hunk 1 above)
Comment (nqhhdev) [reply docs/adr/0022-fix-double-vertical-scroll-bar-on-composer.md:18]: That padding distance is equal to the button bar, if you don't remove it, you will not be able to see the automatic line breaks.

**Comment by dab246** [comment]
Please squash commit

**Pull request merged**